src/main/kotlin/no/nav/familie/ks/sak/kjerne/brev/begrunnelser/Begrunnelse.kt

```diff
@@ -147,6 +147,22 @@ enum class Begrunnelse : IBegrunnelse {
         override val begrunnelseType = BegrunnelseType.AVSLAG
         override val sanityApiNavn = "avslagBosattIRiket"
```

Comment:
Ser bra ut! 🙌 

Ref diskusjon på slack, har du mulighet til å fjerne de to andre avslagtekstene som ligger igjen (AVSLAG_UREGISTRERT_BARN og AVSLAG_BOSATT_I_RIKET)?

Reply:
Tok meg friheten til å bare gjøre dette @charliemidtlyng!